### PR TITLE
Support for JDA 5.0.0-alpha.18 (Message Refactor)

### PR DESCRIPTION
**Pull request by Andre601**
[contributing]: https://github.com/DV8FromTheWorld/JDA/wiki/5%29-Contributing
[pull-request]: https://github.com/JDA-Applications/JDA-Utilities/pulls

#### Pull Request Checklist
Please follow the following steps before opening this PR.<br>
PRs that do not complete the checklist will be subject to denial for
missing information.

- [x] I have checked the [pull request page][pull-request] for upcoming
      or merged features/bug fixes.
- [x] I have read JDA's [contributing guidelines][contributing].

#### Pull Request Information
Check and fill in the blanks for all that apply:

- [ ] My PR fixes a bug, error, or other issue with the library's codebase.
- [x] My PR is for the `command` and `menu` module of the JDA-Chewtils library.
- [ ] My PR creates a new module for the JDA-Chewtils library: `______`.

#### Description

Updates Chewtils to support the latest Message changes made in JDA v5.
Methods using `Message` instances as vars (i.e. `reply(Message)`) have been updated to use `MessageCreateData` instances instead.

`respond(File, String, AttachmentOption...)` was changed to `respond(File, String, String, boolean)` due to `AttachmentOption` being removed.
The new method now uses the second String for the description (Set to null or empty for no description) and the boolean to mark as spoiler.

I'll leave this PR as a draft until the next Alpha is dropped by JDA that implements it completely, so that we have a proper version to support and not just a commit.
(I mainly made this change to use it in my bot :P)

### CODE REVIEW

**Comment by Chew** [comment]
I started on this but gave up, thanks for takin over 😄

**Comment by Andre601** [comment]
Guess the new version requires to propagate first...

**Comment by Chew** [comment]
Yeah it can take an hour or so, welcome to central

**Comment by Chew** [comment]
Nevermind, that's not the issue 😄

File: build.gradle [hunk 1]
```diff
@@ -36,15 +36,15 @@ allprojects {
...
+        jdaVersion = '5.0.0-alpha.18'
         slf4jVersion = '1.7.36'
         okhttpVersion = '4.9.3'
         findbugsVersion = '3.0.2'
         jsonVersion = '20220320'
         junitVersion = '4.13.1' // TODO Move to junit 5?
 
         dependencies {
-            jda = { [group: 'net.dv8tion', name: 'JDA', version: jdaVersion] }
+            jda = { [group: 'com.github.DV8FromTheWorld', name: 'JDA', version: jdaVersion] }
```
Comment (Chew) [inline build.gradle]:
```suggestion
            jda = { [group: 'net.dv8tion', name: 'JDA', version: jdaVersion] }
```

**Review by Chew — APPROVED** [review]

**Review by Chew — APPROVED** [review]

**Pull request merged**